src/observer/management/commands/get_profiles_count.py

```diff
@@ -0,0 +1,22 @@
+import sys, json
```

Comment:
`get` can be counter-intuitive for a command that does not just read the API but also stores values in the database